openvdb/openvdb/tools/PotentialFlow.h

```diff
@@ -17,7 +17,6 @@
 #include "Mask.h" // interiorMask
 #include "Morphology.h" // erodeActiveValues
 #include "PoissonSolver.h"
-#include <openvdb/openvdb.h>
```

Comment:
I haven't taken a close look here but what's the reason for removing this include? I'm a strong proponent for IWYU.

Reply:
It's a double include at line [13 and 20](https://github.com/AcademySoftwareFoundation/openvdb/pull/1418/files/978961ca2d11a6d61707596338fca2b3cbff38f7#diff-f3ebcfe3981bcdd8af15efe8f842133e9e98d510e0b6fa5edf42542cfce0f6d1L13-L20).

Reply:
Ah, thank you!